resources/compass/values.yaml

```diff
@@ -4,23 +4,23 @@ global:
       path: eu.gcr.io/kyma-project/incubator
     connector:
       dir:
-      version: 1.4.0
+      version: 1.4.1
```

Comment:
I am not sure if you can bump images of `compass` as you do not create release branch for it, so the images will not be built.

Reply:
The PR is already created on compass repo, so all images have been built.

Reply:
@Szymongib Please take another look now :)